src/lib/libwebaudio.js

```diff
@@ -294,8 +300,8 @@ let LibraryWebAudio = {
       numberOfOutputs: HEAP32[options+1],
       outputChannelCount: HEAPU32[options+2] ? readChannelCountArray(HEAPU32[options+2]>>2, HEAP32[options+1]) : void 0,
       processorOptions: {
-        'cb': callback,
-        'ud': userData,
+        'cb': {{{ toIndexType('callback') }}},
+        'ud': {{{ toIndexType('userData') }}},
```

Comment:
And maybe do this at the in audio_worklet when `this.userData` is assigned?

Reply:
I did originally but it needed doing in multiple places. Let me take another look (since I got the hang of the changes required).

Reply:
The idea is that we want to pass pointers around as just number and only convert to/from BigInt values when, fir example, directly accessing the table

Reply:
Understood, I'll make the change. I've a bunch of changes incoming since adding `GLOBAL_BASE=2gb` (the existing tests weren't catching the failures).